### DISCUSSION TITLE
Bug ou Non

### DISCUSSION THREAD

**Discussion by ghost** [mak-gitdev/HA_enoceanmqtt#45] [General]
Hello

alors pour commence , super boulot @mak-gitdev 

Je teste enocean , avec des nodon fil pilote 

Par contre j'ai deux lignes qui reste sur inconnu 
LC et OC 

Et serais t(il possible de savoir a quoi sert les ligne qui parle de query , car la je suis largué !! 
merci

![1](https://user-images.githubusercontent.com/103725660/220665700-ec51621d-5e16-4716-9314-3d00caa7f7c0.png)
![2](https://user-images.githubusercontent.com/103725660/220665721-b384b3a1-9353-4cd2-8fa5-fa6d77053f85.png)

**Comment by mak-gitdev** [comment]
Salut @Sonic62,

Ce n'est pas une erreur. Ces deux infos sont des statuts remontés par ton appareil.
Mais les statuts ne sont pas remontés automatiquement, il faut les demander à l'appareil (Query), d'où les boutons Query Status/Power/Energy.

Par contre pour garder une uniformité et pour que les infos servent au plus grand nombre, je préfère pour l'instant que le GitHub reste en Anglais :sweat_smile:
A voir si ça changera.
Si tu as un compte HACF, mon pseudo est @mak-dev.
N'hésite pas à me contacter par message privé si besoin.
